addons/binding/org.openhab.binding.nibeuplink/ESH-INF/thing/vvm320-thing.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="nibeuplink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="vvm320">
+		<label>NibeUplink</label>
+		<description>data retrieved from Nibe Uplink web interface</description>
+
+		<channel-groups>
+			<channel-group typeId="vvm320-sensors" id="sensor" />
+			<channel-group typeId="vvm320-settings" id="setting" />
+		</channel-groups>
+
+		<config-description-ref uri="thing-type:nibeuplink:web" />
+	</thing-type>
+
+	<thing-type id="vvm320-sensors">
+		<label>NibeUplink</label>
+		<description>data retrieved from Nibe Uplink web interface</description>
+
+		<channel-groups>
+			<channel-group typeId="vvm320-sensors" id="sensor" />
+		</channel-groups>
+
+		<config-description-ref uri="thing-type:nibeuplink:web" />
+	</thing-type>
+
+	<thing-type id="vvm320-settings">
+		<label>NibeUplink</label>
+		<description>data retrieved from Nibe Uplink web interface</description>
+
+		<channel-groups>
+			<channel-group typeId="vvm320-settings" id="setting" />
+		</channel-groups>
+
+		<config-description-ref uri="thing-type:nibeuplink:web" />
+	</thing-type>
+
+	<thing-type id="vvm320-special">
+		<label>NibeUplink</label>
+		<description>data retrieved from Nibe Uplink web interface</description>
+
+		<channel-groups>
+            <channel-group typeId="vvm320-general" id="general" />
```

Comment:
Indentation is incorrect for all 4 channel-group's

Reply:
fixed it